time/src/formatting/formattable.rs

```diff
@@ -11,7 +11,12 @@ use crate::formatting::{
 };
 use crate::{error, Date, Time, UtcOffset};
 
-/// A type that can be formatted.
+/// A type that describes a format.
+///
+/// Implementors of [Formattable] are [format descriptions](crate::format_description).
```

Comment:
```suggestion
/// Implementers of [`Formattable]` are [format descriptions](crate::format_description).
```

Spelling and putting types/traits in inline code blocks. The latter should be done in other places as well.

Reply:
Rustdoc generates "Implementors" with an 'o': e.g. <https://docs.rs/time/latest/time/formatting/trait.Formattable.html#implementors> and it is the spelling used in the language documentation.

Will update the code blocks.

Reply:
Fair enough on the spelling, even if that's not how I would spell it.

Reply:
Heh, looks like you blindly accepted the suggested change I had. I mistyped it — the trailing backtick is in the wrong location. That will need to be fixed, with the other locations also receiving backticks as appropriate. CI should also pass.